projects/detr3d/configs/nuscene/detr3d_res101_gridmask.py

```diff
@@ -0,0 +1,221 @@
+_base_ = [
+    '../../../../configs/_base_/datasets/nus-3d.py',
+    '../../../../configs/_base_/default_runtime.py'
+]
+
+custom_imports = dict(imports=['projects.detr3d.mmdet3d_plugin'])
```

Comment:
Most of the comments are the same as https://github.com/open-mmlab/mmdetection3d/pull/2170. Please refine this PR like https://github.com/open-mmlab/mmdetection3d/pull/2170.